package.json

```diff
@@ -15,17 +15,17 @@
   "keywords": [
     "who's on first"
   ],
-  "author": "mapzen",
-  "license": "MIT",
   "dependencies": {
     "lodash": "^3.10.1",
+    "pelias-config": "^1.0.3",
     "pelias-logger": "0.0.8",
+    "pelias-parallel-stream": "0.0.1",
+    "pelias-model": "git://github.com/pelias/model.git#c97cf830d4fe9cb8333f714ee70a72b476779e9c",
```

Comment:
Ah missed that! Just seemed like something that didn't belong in dev-dependencies. :grimacing: Good catch.